_posts/2020-06-04-App Service Continuous Deployment for Windows Containers with GitHub Actions.md

```diff
@@ -0,0 +1,403 @@
+---
+title: "Continuous Deployment for Windows Containers with GitHub Actions"
+author_name: "Jeff Martinez"
+toc: true
+toc_sticky: true
+tags:
+    - dotnet
+    - windows containers
+---
+
+Github Actions enables you to easily automate any part of your development workflow. GitHub Actions are defined as YAML files in the `.github/workflows` directory of your repository. The workflows are triggered by an event, such as a push to a specific branch, a comment on a Pull Request, or on a CRON schedule.
+
+In this article, we will use GitHub Actions for Azure to deploy a Windows Container application to App Service. The sample application is already configured to be used in a Windows Container app, pushed to a registry, and deployed to App Service. Of course, you can use this guide to add the correct deployment tasks to work with your own applications.
+
+## Sample Application
+
+> If you would like to get started with your own application, you can skip to the next section.
+
+The sample application is a simple task-tracking app built with .NET Framework using Azure SQL for storage.  The project has a workflow file, *main.yaml*, that is set up for continuous deployment. You have your choice of using Azure Container Registry (ACR) or Docker Hub for your registry needs (the difference in syntax will be explained below).
+
+Find the full repository samples for [.NET Framework](https://aka.ms/dotnetframeworkdeployment) and [.NET Core](https://aka.ms/dotnetcoredeployment) at these highlighted links.
+
+1. Clone the repository
+1. Add in the necessary GitHub secrets so actions knows where to connect to your Azure resources and committing your changes to the master branch will trigger the build.  
+
+## Create Resources
+
+Create the following resources. You will need information from each resource that will be used in the file and stored in your secrets. Create your choice of registry (Azure Container Registry or Docker Hub) first since you will need information from there before you can create your App Service.
+
+1. Azure Container Registry or a Docker Hub container registry
+1. App Service (Web App for Container)
```

Comment:
I think this needs some clarity around the inputs on the Web App create flow. Which inputs are required, and which they don't need to worry about. (OS, Publish, SKU, everything in the "Docker" tab)

Reply:
This will require some restructuring of the Secure Secrets section to be more explicit here.  I'll be making another commit with changes/additions